ec2/ec2-create-instance.sh

```diff
@@ -201,5 +205,12 @@ CLICKABLE_LINK="http://${PUBLIC_DNS}"
 echo "To ssh into the new instance:"
 echo "ssh -i $PEM_FILE $USER_AT_HOST"
 echo "Branch $BRANCH from $REPO_URL has been deployed to $CLICKABLE_LINK"
-echo "When you are done, please terminate your instance with:"
-echo "aws ec2 terminate-instances --instance-ids $INSTANCE_ID"
+
+DESTROY_CMD="aws ec2 terminate-instances --instance-ids $INSTANCE_ID"
+if [ -z "$DESTROY" ]; then
+   echo "When you are done, please terminate your instance with:"
+   echo "$DESTROY_CMD"
```

Comment:
This looks find but I'm wondering if this message could also be show above just after this line:

`echo "giving instance 60 seconds to wake up..."`

In the past anyway, the script would sometime succeed in spinning up an instance (that should be terminated) but fail to ssh in and run all the Ansible code. So you end up with a mostly blank instance that needs to be terminated. It would be convenient if the "please terminate your instance with:" message appeared above as well.